docs/tables/aws_waf_traffic_log/queries.md

```diff
@@ -0,0 +1,325 @@
+  sum(case when captcha_response.failureReason = 'TOKEN_INVALID' then 1 else 0 end) as challenge_token_invalid,
+  sum(case when captcha_response.failureReason = 'TOKEN_INVALID' then 1 else 0 end) as captcha_token_invalid,
+  sum(case when captcha_response.failureReason = 'TOKEN_DOMAIN_MISMATCH' then 1 else 0 end) as challenge_token_domain_mismatch,
+  sum(case when captcha_response.failureReason = 'TOKEN_DOMAIN_MISMATCH' then 1 else 0 end) as captcha_token_domain_mismatch,
+  sum(case when captcha_response.failureReason = 'TOKEN_EXPIRED' then 1 else 0 end) as challenge_token_expired,
+  sum(case when captcha_response.failureReason = 'TOKEN_EXPIRED' then 1 else 0 end) as captcha_token_expired,
+  sum(case when captcha_response.failureReason = 'TOKEN_MISSING' then 1 else 0 end) as challenge_token_missing,
+  sum(case when captcha_response.failureReason = 'TOKEN_MISSING' then 1 else 0 end) as captcha_token_missing
+from 
+  aws_waf_traffic_log
+group by 
+  client_ip
+order by 
+  total_requests desc;
+```
+
+## Operational Examples
+
+### Retrieve terminating rule matched data for requests
+Extract details of requests that were terminated due to rule matches, helping analyze why a request was blocked or allowed.
+
+```sql
+with terminating_rule_match_details as (
+  select
+    timestamp,
+    http_request.clientIp as client_ip,
+    http_request.uri as request_uri,
+    action,
+    unnest(from_json(terminating_rule_match_details, '["JSON"]')) as match_details
+  from
+    aws_waf_traffic_log
+  where
+    json_array_length(terminating_rule_match_details) > 0
+)
+select
+  timestamp,
+  client_ip,
+  request_uri,
+  action,
+  match_details ->> 'conditionType' as condition_type,
+  match_details ->> 'sensitivityLevel' as sensitivity_level,
+  match_details ->> 'location' as location,
+  match_details ->> 'matchedData' as matched_data
+from
+  terminating_rule_match_details;
+```
+
+### Identify requests missing labels
+
+Retrieve requests that do not contain labels, which help in categorizing and identifying the reason for request handling.
+
+```sql
+select
+  timestamp,
+  action,
+  http_request.clientIp as client_ip,
+  request_headers_inserted
+from
+  aws_waf_traffic_log
+where
+  labels is null
+order by
+  timestamp;
+```
+
+### Analyze blocked requests by rule
+
+Retrieve the number of requests blocked by each rule, providing insights into which rules are most frequently triggered.
+
+```sql
+with blocked_rule as (
+  select
+    timestamp,
+    unnest(
+      from_json(rule_group_list, '["JSON"]')
+    ) as rule_group,
+    action,
+    http_request
+  from
+    aws_waf_traffic_log
+   where
+    action = 'BLOCK'
+)
+select
+  timestamp,
+  action,
+  http_request.clientIp as client_ip,
+  http_request.uri,
+  (rule_group -> 'terminatingRule' ->> 'ruleId') as terminating_rule_id,
+  (rule_group -> 'terminatingRule' ->> 'action') as terminating_rule_action
+from
+  blocked_rule;
+```
+
+### Identify the most targeted URLs
```

Comment:
Queries that use `limit 10`, or similar limits, should have that reflected in the title/description